### ISSUE TITLE
FbMetaData.xml CompositeIdentifierSeparatorPattern value incorrect

### ISSUE THREAD

**Issue by BlackbirdSQL** [FirebirdSQL/NETProvider#1178]
The FbMetaData.xml CompositeIdentifierSeparatorPattern value is a Regex pattern and needs to be escaped.

**Comment by cincuranet** [comment]
Yup. Do you care to create PR?

**Comment by BlackbirdSQL** [comment]
Yes. As soon as I get a gap I'll do that and the FbDatabaseInfo change at the same time.
Shipping EntityFramework.Firebird with the extension and preventing the IDE from accessing it in a user's projects is working great btw. No more invalid cast between assembly [A] and assembly [B].
It means I can remain up to date with your releases without it creating issues when the user's versions are out of date.
Took some doing because VS constantly updates it's dependency tables on a hundred and one different events.
